CHANGELOG.unreleased.md

```diff
@@ -12,3 +12,5 @@ Add here any changes made in a PR that are relevant to end users. Allowed sectio
 Section headings should be at level 3 (e.g. `### Added`).
 
 ## Unreleased
+
+- fixed some charaters([?\*) would cause file uploads to fail with wandb-core (@jacobromero in https://github.com/wandb/wandb/pull/9475)
```

Comment:
```suggestion
- fixed some characters ([?\*) would cause file uploads to fail with wandb-core (@jacobromero in https://github.com/wandb/wandb/pull/9475)
```

- This changelog entry is not very clear.
- Could you specify that you fixed an issue with media not being handled properly when the file names included glob? characters.